engine/baml-schema-wasm/src/runtime_wasm/mod.rs

```diff
@@ -1265,7 +1305,9 @@ impl WasmFunction {
         rt: &WasmRuntime,
         params: JsValue,
     ) -> Result<WasmPrompt, wasm_bindgen::JsError> {
-        let params = serde_wasm_bindgen::from_value::<BamlMap<String, BamlValue>>(params)?;
+        let mut params = serde_wasm_bindgen::from_value::<BamlMap<String, BamlValue>>(params)?;
+        let node_index = params.shift_remove("node_index");
```

Comment:
this breaks a user who defines a fn with node_index as an arg

Reply:
Will fix tmrw morning